catalog/format/iceberg/src/test/java/org/projectnessie/catalog/formats/iceberg/nessie/TestNessieModelIceberg.java

```diff
@@ -316,6 +326,65 @@ public void icebergNessieTypeConversion(IcebergType icebergType) throws Exceptio
         .isEqualTo(nessieIdHasher("NessieTypeSpec").hash(nessieAgain).generate());
   }
 
+  @Test
+  public void icebergNested() throws Exception {
```

Comment:
Maybe add a field of type Map? It doesn't look like we are testing this specific type.